jbpm-wb-integration/jbpm-wb-integration-backend/src/test/java/org/jbpm/workbench/wi/backend/server/dd/DDEditorServiceTest.java

```diff
@@ -262,6 +262,36 @@ public void testValidateMvelResolver() {
     }
 
     @Test
+    public void testValidateMvelResolverWithSupportedParameters() {
+        List<NamedObjectModel> models = Arrays.asList(new NamedObjectModel[]{
+                new NamedObjectModel(ItemObjectModel.MVEL_RESOLVER,
+                                     "item-name",
+                                     "java.util.Arrays.asList(ksession, taskService, runtimeManager, classLoader, entityManagerFactory);")});
+
+        List<ValidationMessage> validationMessages = validateObjectModels(null, models);
+        assertEquals(0, validationMessages.size());
+
+    }
+
+    @Test
+    public void testValidateMvelResolverWithUnSupportedParameters() {
+        List<NamedObjectModel> models = Arrays.asList(new NamedObjectModel[]{
+                new NamedObjectModel(ItemObjectModel.MVEL_RESOLVER,
+                                     "item-name",
+                                     "java.util.Arrays.asList(unsupportedParamName);")});
+
+        List<ValidationMessage> validationMessages = validateObjectModels(null, models);
+        assertEquals(1, validationMessages.size());
+
+        ValidationMessage error = validationMessages.get(0);
+        assertEquals(Level.WARNING, error.getLevel());
+        assertTrue(error.getText().startsWith("Could not compile mvel expression"));
```

Comment:
I'm not 100 % sure here without looking at the Javadoc, but I think that `Assertions.assertThat(error.getText()).startsWith(...)` should work as well.